middleware/logging.go

```diff
@@ -62,14 +63,17 @@ func logRequest(metric *prometheus.SummaryVec) func(httprouter.Handle) httproute
 
 			next(wrapped, r, ps)
 			duration := time.Since(start)
-			log.LogNoRequestID("received HTTP request",
-				"remote", r.RemoteAddr,
-				"proto", r.Proto,
-				"method", r.Method,
-				"uri", r.URL.RequestURI(),
-				"duration", duration,
-				"status", wrapped.status,
-			)
+
+			if glog.V(4) && (wrapped.status < 400) {
```

Comment:
Shouldn't we change this to `if glog.V(4) || (wrapped.status >= 400)` so that we will still log errors?

Reply:
You're right. Fixed.